source/sync/configure/enable-sync.txt

```diff
@@ -236,49 +252,82 @@ Enable Partition-Based Sync
 
             Once you've determined how to decide a user's read and write permissions for a
             given partition, define the corresponding JSON expressions in the
-            ``read`` and ``write`` fields of ``config.sync.partition.permissions``.
+            ``read`` and ``write`` fields of ``partition.permissions``, similar to:
 
             .. code-block:: json
-               :emphasize-lines: 9-14, 15-17
+               :emphasize-lines: 7-15
 
                {
-                 "config": {
-                   "sync": {
-                     "state": "enabled",
-                     "partition": {
-                       "key": "owner_id",
-                       "type": "string",
-                       "permissions": {
-                         "read": {
-                           "$or": [
-                             { "%%user.id": "%%partition" },
-                             { "%%user.custom_data.shared": "%%partition" }
-                           ]
-                         },
-                         "write": {
-                           "%%user.id": "%%partition"
-                         }
-                       }
+                 ...
+                 "partition": {
+                   "key": "owner_id",
+                   "type": "string",
+                   "permissions": {
+                     "read": {
+                       "$or": [
+                         { "%%user.id": "%%partition" },
+                         { "%%user.custom_data.shared": "%%partition" }
+                       ]
+                     },
+                     "write": {
+                       "%%user.id": "%%partition"
                      }
-                   },
-                   ...
-                 },
+                   }
+                 }
                  ...
                }
 
+         .. step:: Specify Values for Sync Optimization
+
+            Sync provides features that enable you to optimize Sync 
+            performance and improve the client data recovery process. These 
```

Comment:
```suggestion
            performance and improve the data recovery process on the client. These 
```

Suggestion/Not Required for LGTM:
The phrase "client data recovery process" sounds a bit less natural than "data recovery process on the client" even if they are similar.